pkgs/development/compilers/rust/cargo-auditable.nix

```diff
@@ -37,4 +42,12 @@ in
 
 rustPlatform.buildRustPackage.override { cargo-auditable = bootstrap; } (args // {
   auditable = true; # TODO: remove when this is the default
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    installManPage cargo-auditable/cargo-auditable.1
+  '';
```

Comment:
I don't know if it matters if this is in the bootstrap version or not. Probably not to much.